velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -466,11 +476,20 @@ class ExpressionFuzzer {
 
   std::vector<core::TypedExprPtr> generateArgs(const CallableSignature& input) {
     std::vector<core::TypedExprPtr> inputExpressions;
-    inputExpressions.reserve(input.args.size());
+    auto variableArity = input.variableArity;
+    auto numVarArgs = !variableArity
+        ? 0
+        : folly::Random::rand32(FLAGS_max_num_varargs + 1, rng_);
+    inputExpressions.reserve(input.args.size() + numVarArgs);
 
     for (const auto& arg : input.args) {
       inputExpressions.emplace_back(generateArg(arg));
     }
+    if (variableArity) {
```

Comment:
nit: for readability, I'd move the variableArity check and resizing the input expressions vector to this block, but not a big deal.

Reply:
I removed the condition but I did not change it to vector.resize() because that will make the rest of the varargs the same.